apps/pixeloven/site/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "@pixeloven/site",
+  "version": "6.0.0-beta.7",
+  "description": "Example project using the framework",
+  "homepage": "https://github.com/pixeloven/pixeloven",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/pixeloven/pixeloven.git"
+  },
+  "author": "Brian Gebel",
+  "license": "MIT",
+  "private": true,
+  "dependencies": {
+    "@pixeloven-react/routing": "^6.0.0-beta.7",
+    "classnames": "2.2.6",
+    "express": "4.17.1",
+    "express-winston": "3.2.1",
+    "react": "16.8.6",
+    "react-dom": "16.8.6",
+    "react-helmet": "6.0.0-beta",
+    "react-router": "5.0.1",
+    "react-router-dom": "5.0.1",
+    "winston": "3.2.1"
+  },
+  "devDependencies": {
+    "@pixeloven/cli": "^6.0.0-beta.7",
+    "@pixeloven/cli-addon-generators": "^6.0.0-beta.7",
+    "@pixeloven/cli-addon-storybook": "^6.0.0-beta.7",
+    "@pixeloven/cli-addon-webpack": "^6.0.0-beta.7",
+    "@types/classnames": "2.2.9",
+    "@types/enzyme": "3.10.3",
+    "@types/enzyme-adapter-react-16": "1.0.5",
+    "@types/express": "4.17.0",
+    "@types/express-winston": "3.0.0",
+    "@types/react": "16.8.16",
+    "@types/react-dom": "16.8.4",
+    "@types/react-helmet": "5.0.8",
+    "@types/react-router": "5.0.3",
+    "@types/react-router-dom": "4.3.4",
+    "@types/supertest": "2.0.7",
+    "@types/webpack": "4.39.2",
+    "enzyme": "3.10.0",
+    "enzyme-adapter-react-16": "1.14.0",
+    "lint-staged": "9.2.5",
+    "supertest": "3.3.0"
+  },
+  "devEngines": {
+    "node": "8.x || 9.x || 10.x"
+  },
+  "scripts": {
+    "build:development": "NODE_ENV=production pixeloven webpack build --stats --source-map --path /",
```

Comment:
this env should be development

Reply:
Nope needs to be production. This is the build and like SSR the build should always be done in prod mode. This is not the target environment but instead the mode the node is running.